<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__set_italic">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__set_underline">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__set_font">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__set_red_color">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Red Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__append">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Append</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__clear">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__set_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SetText</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__print">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__size_up">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Up</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#ffaa7f;&quot;&gt;Text Edit&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; color:#aaaaff;&quot;&gt;supports Rich Text&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="text_edit__size_down">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Size Down</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plain_text_edit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Plain Text Edit
supports Plain Text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plain_text_edit__print">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plain_text_edit__clear">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="plain_text_edit__set_text">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SetText</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plain_text_edit__append">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Append</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
